data/autoyast_sle15/autoyast_scc_up_ppc64le.xml

```diff
@@ -87,7 +67,7 @@
           <partition_nr config:type="integer">2</partition_nr>
           <partition_type>primary</partition_type>
           <resize config:type="boolean">false</resize>
-          <size>30054285312</size>
+          <size>35G</size>
```

Comment:
It seems the autoyast format for profile updated, just wondering why set as 35G while not 30G or 40G.

Reply:
This also needed by powerVM autoyast migration test. For other powerVM test cases, it sets 35G for hard disk